pyresample/geometry.py

```diff
@@ -1618,7 +1618,7 @@ def colrow2lonlat(self, cols, rows):
         p = Proj(self.proj_str)
         x = self.projection_x_coords
         y = self.projection_y_coords
-        return p(y[y.size - cols], x[x.size - rows], inverse=True)
+        return p(x[cols],  y[rows], inverse=True)
```

Comment:
Any idea why the inverse was used in the first place (the `y.size - ` part I mean)? Shouldn't that still be used?

Reply:
I think it's because of the test area definition it was using before, plus the fact that it was mixing `x` and `y` with `cols` and `rows`. In that specific case, the `x` and `y` are the inverse of each other, so actually `y[y.size - cols]==x[cols]` holds, and vice versa.

Reply:
I *think* that makes sense to me. If the old tests still pass then I'm ok with it.

Reply:
Yep they do, I actually just reintroduced them!